### PR TITLE
Add styling to the react app

### PR DESCRIPTION
**Pull request by Atifzada**
### In this PR following changes are made:

- This is the last milestone of this week's project and I have managed to complete the styling of the bookstore app by following the Zeplin template

### CODE REVIEW

File: src/components/styles/Books.css
Comment (franclobo) [inline src/components/styles/Books.css:1]: - [ ] Well done doing your own styles, but please 🙏🏽 correct all the styles such as font-family, font-size, color, margins and paddings to match with the Zeplin design, it is a project requirement to approve this milestone 👇🏽 

| Zeplin | Your design |
|---|---|
| image | image |

File: src/components/styles/Navigation.css [hunk 1]
```diff
@@ -1,21 +1,64 @@
...
 }
 
-.nav ul li {
+li,
+a {
   list-style-type: none;
+  text-decoration: none;
+  color: #121212;
+  font-size: 20px;
 }
```
Comment (franclobo) [inline src/components/styles/Navigation.css:20]: - [x] Please 🙏🏽 correct the styles according to the font family specifications to match the Zeplin design.

image

File: src/components/styles/Books.css [hunk 2]
```diff
@@ -1,11 +1,162 @@
...
+.booksTitle {
+  font-family: serif;
+  font-size: 1.775rem;
+  font-weight: bolder;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -1.2px;
+  color: #121212;
+}
```
Comment (franclobo) [inline src/components/styles/Books.css]: - [x] Please 🙏🏽 correct the styles according to the font family specifications to match the Zeplin design.

image

File: src/components/styles/Books.css [hunk 3]
```diff
@@ -1,11 +1,162 @@
...
+  font-family: "RobotoSlab", serif;
+  font-size: 0.975rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  margin: 0;
+}
```
Comment (franclobo) [inline src/components/styles/Books.css:56]: - [x] Please 🙏🏽 correct the styles according to the font family specifications to match the Zeplin design.

image

File: src/components/styles/NewBook.css [hunk 4]
```diff
@@ -7,4 +7,77 @@ form {
...
+  margin: 5%;
+}
+
+.formTitle {
+  font-size: 1.25rem;
+  font-weight: 700;
+  letter-spacing: -0.18px;
+  color: #888;
+  margin-left: 5%;
+}
```
Comment (franclobo) [inline src/components/styles/NewBook.css:34]: - [x] Please 🙏🏽 correct the styles according to the font family specifications to match the Zeplin design.
image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Atifzada, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct Git flow ✔️ 
- Descriptive PR ✔️ 
- Good styles ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/styles/NewBook.css (hunk 4 above)
Comment (Atifzada) [reply src/components/styles/NewBook.css:34]: Thank you for the helpful review. Surely I will fix these issue

**Review by rotimiazeez — APPROVED** [review]
Hi @Atifzada 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job implementing the requested changes 👍🏽
- Good job adjusting the styling of your app 👍🏽 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**